### PR TITLE
vq 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by vqmind**
Easy to use utility for querying json content.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/vq.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  license "MIT"
+
+  def install
+    inreplace "brew/vq", "##PREFIX##", prefix.to_s
+    prefix.install "app-1.0.0.jar"
+    bin.install "brew/vq"
+  end
+
+  test do
+    assert_match "seattle", shell_output(" echo '{\"city\":\"seattle\"}' | #{bin}/vq \"$.city\" ")
```
Comment (SMillerDev) [inline Formula/vq.rb:15]:
```suggestion
    assert_match "seattle", pipe_output("{\"city\":\"seattle\"}", "#{bin}/vq \"$.city\" ")
```

File: Formula/vq.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Vq < Formula
+  desc "Utility for querying json"
+  homepage "https://vqmind.github.io/vq/"
+  url "https://github.com/vqmind/vq/releases/download/1.0.0/app-1.0.0.tar.gz"
+  sha256 "057d570bd3a9522f6e392c3a5dbbf412d4cc676b16f752626b9acdc06438f3d3"
+  license "MIT"
+
+  def install
+    inreplace "brew/vq", "##PREFIX##", prefix.to_s
+    prefix.install "app-1.0.0.jar"
```
Comment (SMillerDev) [inline Formula/vq.rb:10]:
```suggestion
    prefix.install "app-#{version}.jar"
```

**Review by SMillerDev — COMMENTED** [review]
Doesn't the presence of a `.jar` file mean it depends on openjdk?

File: Formula/vq.rb (hunk 2 above)
Comment (vqmind) [reply Formula/vq.rb]: Done. Thanks

**Comment by vqmind** [comment]
Fixed with depends_on. Thanks.

**Comment by vqmind** [comment]
Thanks for quick response. I've fixed the comments. Kindly let me know if anything else i can do.

File: Formula/vq.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  depends_on "openjdk"
+
+  def install
+    prefix.install "app-#{version}.jar"
+
+    (bin/"vq").write <<~EOS
+      #!/bin/bash
+      exec "#{Formula["openjdk"].opt_bin}/java" -jar #{prefix}/app-#{version}.jar "$@"
+    EOS
```
Comment (SMillerDev) [inline Formula/vq.rb:16]:
```suggestion
    libexec.install "app-#{version}.jar" => "vq.jar"
    bin.write_jar_script libexec/"vq.jar", "vq"
```

**Comment by bayandin** [comment]
```
  vq:
    * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Pull request closed without merging**